libs/npm/core/src/components/grid-table/GridTable.tsx

```diff
@@ -191,6 +192,8 @@ export const GridTable = <T extends object = Record<string, unknown>>({
     onChangeSort(sortBy);
   }, [onChangeSort, sortBy]);
 
+  const { id } = useParams();
```

Comment:
What does the `id` do for the GridTable?  This looks like contamination of purpose.

Reply:
Is every single GridTable going to use this functionality?  I doubt it.  Abstract it so that it makes sense for general use.